extensions/blocks/opentable/attributes.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { compact, reduce } from 'lodash';
+
+const optionValues = options => options.map( option => option.value );
+
+export const languageOptions = [
+	{ value: 'en-US', label: __( 'English-US', 'jetpack' ) },
```

Comment:
Do we need to `__()` these labels?
They indicate language names, and are already translated.

Reply:
Good point, that doesn't make sense!